addons/point_of_sale/static/tests/tours/helpers/PaymentScreenTourMethods.js

```diff
@@ -277,3 +277,29 @@ export function clickShipLaterButton() {
         },
     ]
 }
+
+export function addShippingDate(){
+    return [
+        {
+            content: 'Choose ship later',
+            trigger: '.payment-buttons .button.btn:contains("Ship Later")'
+        },
+        {
+            content: 'Confirm date',
+            trigger: '.o-main-components-container .button.confirm'
+        }
+    ];
+}
+
+export function addCustomer(){
```

Comment:
You can already select the customer from the product screen. No need to add new methods here